### DISCUSSION TITLE
SIXEL

### DISCUSSION THREAD

**Discussion by bbs-addict** [mkrueger/icy_term#8] [Ideas]
https://en.wikipedia.org/wiki/Sixel

I think with some slight improvements maybe you could add this.  I have 2 images on my local bbs. Just me only.
These are all on the link for wikipedia:

https://sourceforge.net/projects/sixelviewer/

Thanks 
DaviD S

**Comment by mkrueger** [comment]
Sixels are implemented. 
However I fixed some sixel related bugs lately (in 0.5.1).

If there are any real world examples that don't work - I'm sure there are - open an issue for that.
